src/FormBuilder/resources/js/form-builder/src/styles/_block-editor.scss

```diff
@@ -116,6 +117,17 @@
 .block-editor-block-list__block {
     position: relative;
 
+    .wp-block {
+
+        &.is-selected {
+            padding: 1.5rem;
+            border-style: solid;
+            border-width: thin;
+            border-radius: 0.25rem;
+            border-color: var(--wp-admin-theme-color);
+        }
+    }
+
```

Comment:
@glaubersilva did someone mention this as feedback? Let's create a separate PR for this as it affects the form builder UI pretty drastically

Reply:
Yes! @JasonTheAdams ask me to add this focus ring in the blocks, so this change makes this happen:

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/8fac20ca-2be4-401e-963e-23abcf146ce4)

@jdghinson Also did review this focus ring. 

If you notice, I used the same style used in the section border:

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/51c1511b-470e-4202-a571-31da2fe47b65)

So, I'm not sure about how it can affect the UI drastically, maybe I'm missing something.

Reply:
okay got it, with this addition the padding required causes the fields to jump in place when selected which will affect all blocks.  I'm inclined to move this to a new PR so we can examine its change to the UI closer.

Reply:
Resolved in b22dc6d9b04a76b65294e387987b33f71aa76981

New PR here: https://github.com/impress-org/givewp-next-gen/pull/211